target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9531_kuwfi_830d.dts"
```

Comment:
Move the common stuff to a dtsi and create the LED definitions for each board individually to match the LED naming convention.

Reply:
i created 2 separate .dts files